holoviews/tests/operation/testdatashader.py

```diff
@@ -1144,8 +1154,16 @@ def test_points_inspection_dict_streams_instance(self):
         Tap.x, Tap.y = 0.2, 0.3
         inspector = inspect_points.instance(max_indicators=3, dynamic=True, pixels=1,
                                             streams=dict(x=Tap.param.x, y=Tap.param.y))
-        points = inspector(self.img)
+        points = inspector(self.pntsimg)
         self.assertEqual(len(points.streams), 1)
         self.assertEqual(isinstance(points.streams[0], Tap), True)
         self.assertEqual(points.streams[0].x, 0.2)
-        self.assertEqual(points.streams[0].y, 0.3)
\ No newline at end of file
+        self.assertEqual(points.streams[0].y, 0.3)
+
+    def test_polys_inspection_1px_mask(self):
+        if spatialpandas is None:
+            raise SkipTest('Polygon inspect tests require spatialpandas')
+        polys = inspect_polygons(self.polysrgb,
+                                 max_indicators=3, dynamic=False, pixels=1, x=-6, y=5)
+        self.assertEqual(polys.dimension_values('x'), np.array([6, 3, 7, 6]))
+        self.assertEqual(polys.dimension_values('y'), np.array([7, 2, 5, 7]))
```

Comment:
Would write this as:

```suggestion
        self.assertEqual(polys, hv.Polygons([{'x': [6, 3, 7, 6], 'y': [7, 2, 5, 7]}]))
```

Reply:
Don't think that's quite right, still missing the vdim.

Reply:
Sure. Happy to do the asserts that way...

Reply:
Implemented in 7bba4800a